### ISSUE TITLE
Release new version to support Go 1.20

### ISSUE THREAD

**Issue by lcheylus** [mr-karan/doggo#70]
I'm updating my OpenBSD port for doggo with the latest version (0.5.5). I'm using Go 1.20 to build my port (currently supported version on OpenBSD-current / future version 6.4).

But I have an error during my build with Go version 1.20 because github.com/lucas-clemente/quic-go module v0.31.1 (version required for doggo v0.5.5) is not supported:

```
$ go build
(...)
github.com/lucas-clemente/quic-go/internal/qtls
# github.com/lucas-clemente/quic-go/internal/qtls
../../../go/pkg/mod/github.com/lucas-clemente/quic-go@v0.31.1/internal/qtls/go120.go:5:13: cannot use "The version of quic-go you're using can't be built on Go 1.20 yet. For more details, please see https://github.com/lucas-clemente/quic-go/wiki/quic-go-and-Go-versions." (untyped string constant "The version of quic-go you're using can't be built on Go 1.20 yet. F...) as int value in variable declaration
```

The latest version v0.35.1 of quic-go module supports Go 1.20.

Please could you release a new version (0.5.6 ?) of doggo to support build with Go 1.20 ?

**Comment by stefanb** [comment]
quic-go 0.37 was released, supporting Go 1.20 and 1.21: https://github.com/quic-go/quic-go/releases/tag/v0.37.0

**Comment by lcheylus** [comment]
Build OK for release 0.5.6 on OpenBSD/amd64 with Go v1.20 and with Go v1.21 adding commit https://github.com/mr-karan/doggo/commit/9818c32b9db5b4db7d560dfeaa530a1e7bbbcc0e
